### PR TITLE
Fix #196 - handle ChoiceType initialized with Enum

### LINKED ISSUE #196: Enum ChoiceType column results in error
Howdy folks!

The following code produces `TypeError: 'MyEnum' object is not iterable`, but should work. Note that this is also involves package `sqlalchemy_utils`

```py
from enum import Enum

class MyEnum(Enum):
   foo = 1
   bar = 2

class MyModel(Base):
    id = Column(sa.Integer, primary_key=True)
    name = Column(sa.Unicode(255))
    type = Column(ChoiceType(MyEnum, impl=Integer()))

class MyObject(SQLAlchemyObjectType):
    class Meta:
        model = MyModel
```

### PR DESCRIPTION
**Pull request by thejcannon**

### CODE REVIEW

File: graphene_sqlalchemy/converter.py [hunk 1]
```diff
@@ -162,7 +162,12 @@ def convert_enum_to_enum(type, column, registry=None):
 @convert_sqlalchemy_type.register(ChoiceType)
 def convert_column_to_enum(type, column, registry=None):
     name = "{}_{}".format(column.table.name, column.name).upper()
-    return Enum(name, type.choices, description=get_column_doc(column))
+    enum_members = getattr(type.choices, "__members__", None)
+    if enum_members:  # Check if an enum.Enum is used
+        items = [(item.name, item.value) for item in enum_members.values()]
```
Comment (thejcannon) [inline graphene_sqlalchemy/converter.py]: We could also return `Enum.from_enum(type.choices)`.
I don't know enough about what the expected name/description should be to know which way is more correct.

**Comment by thejcannon** [comment]
Closing. I'm going to post a new PR that'll fix this and #199 by modeling the code after `convert_enum_to_enum`

**Comment by thejcannon** [comment]
I reopened as #199 isn't seemingly an issue with this library.

**Comment by thejcannon** [comment]
Closing again in lieu of new PR to fix enums in general

**Comment by squarewave24** [comment]
hello, 
i came across this issue on graphene-sqlalchemy v2.2.2
```
TypeError: cannot unpack non-iterable ReviewStatusEnum object
```
on
```
status = Column(ChoiceType(ReviewStatusEnum, impl=Unicode()), nullable=False)
```

is there a trick to make this work?

**Comment by jnak** [comment]
I'm not familiar with the issue but if you need a quick workaround, you should be able to define your own graphene field, enum and resolver.

**Comment by squarewave24** [comment]
actually it was resolved in latest pre-release version 2.3.* so i am testing that out now.

**Pull request closed without merging**